.github/dependabot.yml

```diff
@@ -14,6 +14,15 @@ updates:
     reviewers: ["khluu", "simon-mo"]
     allow:
       - dependency-type: "all"
+    ignore:
```

Comment:
a comment would be nice here to clarify what the reasoning is for including these here (pinned dependencies that we want to update manually).

It would also be nice to add a note in requirements-common.txt with a note that if we pin new dependencies that we would like to handle manually, that they should be added to this file as well.

Little breadcrumbs like that would be helpful to other people in the future to understand why things are the way they are.